src/Microsoft.AspNetCore.Analyzer.Testing/contentFiles/xunit.runner.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
What's this about?

Reply:
I had to disable appdomains to get DependencyModel to work.